src/appleseed-max-impl/appleseedrenderer/renderersettings.cpp

```diff
@@ -369,7 +369,7 @@ void RendererSettings::apply_settings_to_interactive_config(asr::Project& projec
     params.insert_path("lighting_engine", "pt");
 
     if (m_rendering_threads == 0)
```

Comment:
I wonder if this whole block still makes sense, now that the default for m_rendering_threads is sane (-1). There's no special case for final configurations.

Reply:
For ActiveShade mode I think it still makes sense that when rendering_threads == 0 (e.g. from loading saved scene files) it is always overridden to keep the UI responsive.  
For production rendering mode it is less of an issue as one does not actively manipulate the scene during rendering (as one usually does in ActiveShade mode).

Reply:
Ok.

In that case, the code that sets the number of rendering threads in `apply_common_settings()` could be moved to `apply_settings_to_final_config()`. I think that would make things a little clearer.

Reply:
Yes, absolutely right.